posts/2020-12-18-microprofile-long-running-actions-beta.adoc

```diff
@@ -329,17 +329,17 @@ The  following example shows the `BookHotel` method:
         }
     }
 ----
-The `LRA.Type` value for the `BookHotel` service is been set to MANDATORY, which means that it has to be called as part of an LRA and fails automatically if it called called outside of one. So while the `BookFlight` service can be called outside of an LRA, as it would start its own, the `BookHotel` service returns an error if it is called outside of an LRA.
+The `LRA.Type` value for the `BookHotel` service is set to MANDATORY, which means that it has to be called as part of an LRA and fails automatically if it is called outside of one. So while the `BookFlight` service start its own LRA if called outside of one, the `BookHotel` service returns an error if it is called outside of an LRA.
```

Comment:
We repeat the phase "if it is called outside of one" nearly 3 times in a paragraph so maybe we need to rephrase it?